packages/mux-player/src/media-theme-mux.html

```diff
@@ -447,9 +509,8 @@
     </template>
 
     <template partial="LiveButton">
-      <!-- NOTE: Keeping "seek-live" part for backwards compatibility for now -->
       <media-live-button
-        part="{{section ?? 'top'}} live seek-live button"
+        part="{{section ?? 'top'}} live button"
```

Comment:
chore(blocker): Make sure the docs are appropriately updated to reflect this removal.

Reply:
added https://github.com/muxinc/docs.mux.com/pull/856